ethcore/sync/src/api.rs

```diff
@@ -849,12 +863,13 @@ impl ManageNetwork for LightSync {
 impl LightSyncProvider for LightSync {
 	fn peer_numbers(&self) -> PeerNumbers {
 		let (connected, active) = self.proto.peer_count();
-		let config = self.network_config();
+		let peers_range = self.num_peers_range();
+		debug_assert!(peers_range.end > peers_range.start);
 		PeerNumbers {
 			connected: connected,
 			active: active,
-			max: config.max_peers as usize,
-			min: config.min_peers as usize,
+			max: peers_range.end as usize - 1,
```

Comment:
why is the range exclusive and not inclusive? This is a breaking change. If someone have `min: 10` and `max: 10` specified in the config, the new min and max will be 10 and 9.